Formula/mdxmini.rb

```diff
@@ -59,11 +61,19 @@ def install
     EOS
     system ENV.cc, "mdxtest.c", "-L#{lib}", "-lmdxmini", "-o", "mdxtest"
 
-    result = `#{testpath}/mdxtest #{testpath}/pop-00.mdx #{testpath}`.chomp
+    result = shell_output("#{testpath}/mdxtest #{testpath}/pop-00.mdx #{testpath}").chomp
     result.force_encoding("ascii-8bit") if result.respond_to? :force_encoding
+
     # Song title is in Shift-JIS
-    expected = "\x82\xDB\x82\xC1\x82\xD5\x82\xE9\x83\x81\x83C\x83\x8B         \x83o\x83b\x83N\x83A\x83b\x83v\x8D\xEC\x90\xAC          (C)Falcom 1992 cv.\x82o\x82h. ass.\x82s\x82`\x82o\x81{"
+    # Trailing whitespace is intentional & shouldn't be removed.
+    l1 = "\x82\xDB\x82\xC1\x82\xD5\x82\xE9\x83\x81\x83C\x83\x8B         "
+    l2 = "\x83o\x83b\x83N\x83A\x83b\x83v\x8D\xEC\x90\xAC          "
+    expected = <<~EOS
+      #{l1}
+      #{l2}
+      (C)Falcom 1992 cv.\x82o\x82h. ass.\x82s\x82`\x82o\x81{
+    EOS
     expected.force_encoding("ascii-8bit") if result.respond_to? :force_encoding
```

Comment:
I feel the following may be a bit easier to read:

```
expected = [
             "\x82\xDB\x82\xC1\x82\xD5\x82\xE9\x83\x81\x83C\x83\x8B",
             "\x83o\x83b\x83N\x83A\x83b\x83v\x8D\xEC\x90\xAC",
             " (C)Falcom 1992 cv.\x82o\x82h. ass.\x82s\x82`\x82o\x81{",
           ].join("         ")
```

…

```
assert_equal expected, result
```

___

Or alternatively, inline the `l1` and `l2` constants:

```
expected = <<~EOS
  #{"\x82\xDB\x82\xC1\x82\xD5\x82\xE9\x83\x81\x83C\x83\x8B         "}
  #{"\x83o\x83b\x83N\x83A\x83b\x83v\x8D\xEC\x90\xAC          "}
  (C)Falcom 1992 cv.\x82o\x82h. ass.\x82s\x82`\x82o\x81{
EOS
```

Reply:
If you want to follow up I'm open to the latter, as long as audit doesn't take a shot at it!

Reply:
> as long as audit doesn't take a shot at it!

It does 😒

May as well stay as is, then! 😄

Reply:
Audit likes to get _real punchy_ with whitespace 😸

Reply:
I didn’t even get that far! They pulled me over due to “literal interpolation” already.